pkg/health/health_cert_manager.go

```diff
@@ -13,49 +16,94 @@ func SetDefaultCertificateExpiryWarningPeriod(p time.Duration) {
 	defaultCertExpiryWarningPeriod = p
 }
 
+func GetCertificateRequestHealth(obj *unstructured.Unstructured) (*HealthStatus, error) {
+	var certReq certmanagerv1.CertificateRequest
+	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(obj.Object, &certReq); err != nil {
```

Comment:
If the certificate request is older than 1h and not approved it should be unhealthy